scan/src/components/data-source/DataSourceExecute.re

```diff
@@ -0,0 +1,72 @@
+module Styles = {
+  open Css;
+
+  let container = style([padding2(~h=`px(35), ~v=`px(20))]);
+
+  let paramsContainer = style([display(`flex), flexDirection(`column)]);
+
+  let listContainer = style([marginBottom(`px(25))]);
+
+  let input =
+    style([
+      width(`percent(100.)),
+      background(white),
+      paddingLeft(`px(25)),
+      fontSize(`px(12)),
+      fontWeight(`num(500)),
+      outline(`px(1), `none, white),
+      height(`px(40)),
+      borderRadius(`px(4)),
+      boxShadow(
+        Shadow.box(~inset=true, ~x=`zero, ~y=`zero, ~blur=`px(4), Css.rgba(0, 0, 0, 0.1)),
+      ),
+    ]);
+
+  let buttonContainer = style([display(`flex), flexDirection(`row), alignItems(`center)]);
+
+  let button =
+    style([
+      width(`px(110)),
+      backgroundColor(Colors.btnGreen),
+      borderRadius(`px(6)),
+      fontSize(`px(12)),
+      fontWeight(`num(500)),
```

Comment:
Can you confirm with Q this is the same as the design. I think it looks different but I'm not sure.